xml/Microsoft.VisualC.StlClr/BinaryDelegate`3.xml

```diff
@@ -27,7 +27,7 @@
     </Attribute>
   </Attributes>
   <Parameters>
-    <Parameter Name="" Type="TArg2" Index="0" FrameworkAlternate="netframework-3.5;netframework-4.0" />
+    <Parameter Name="" Type="TArg2" Index="1" FrameworkAlternate="netframework-3.5;netframework-4.0" />
```

Comment:
this change lloks wrong. line 32 already has this.

Reply:
Hmm, you're right, this is not correct ... looks like in this case, the empty parameter name is making the indexing algorithm go wonky :( 

Seems like it's a slightly different issue than https://github.com/mono/api-doc-tools/issues/291, though unfortunately the [original issue there](https://github.com/dotnet/dotnet-api-docs/blob/0012e5ded56afa6e8fd1f359bc62098b949c91b0/xml/System.IO.Compression/DeflateStream.xml#L397) still remains. So I'm going to re-open that bug